net-vpn/tor/tor-0.3.2.9.recipe

```diff
@@ -5,43 +5,51 @@ developers to create new communication tools with built-in privacy features. \
 Tor provides the foundation for a range of applications that allow \
 organizations and individuals to share information over public networks \
 without compromising their privacy."
-HOMEPAGE="https://www.torproject.org"
-COPYRIGHT="
-	2001-2004, Roger Dingledine
-	2004-2006, Roger Dingledine, Nick Mathewson
-	2007-2017, The Tor Project, Inc.
-	"
+HOMEPAGE="https://www.torproject.org/"
+COPYRIGHT="2001-2004 Roger Dingledine
+	2004-2006 Roger Dingledine, Nick Mathewson
+	2007-2018 The Tor Project, Inc."
```

Comment:
On an another note, should we put the most recent holder on top of `COPYRIGHT`? Since that will be what we're going to show our user when they install the hpkg, I believe we would want to place the latest/biggest copyright holder of a project for the user to see.